app/controllers/users_controller.rb

```diff
@@ -1,24 +1,54 @@
 class UsersController < Clearance::UsersController
+  def show
+    @user = User
+            .joins(:profile)
```

Comment:
I prefer 2 space indent for alignment, less noise when changing in the future.
i.e. if you changed `@user` to `@member`, you'd need to re-indent